source/agora/node/Node.d

```diff
@@ -134,16 +134,19 @@ public class Node : API
     /// The first task method, loading from disk, node discovery, etc
     public void start ()
     {
-        log.info("Doing network discovery..");
-        this.network.discover();
-
-        bool isNominating ()
+        this.taskman.runTask(
```

Comment:
https://github.com/bpfkorea/agora/blob/18c86736458a9cc3926f05be4fbf637a98fd8c8c/source/agora/node/Node.d#L544

Reply:
I saw that, but I think I left it in of misunderstanding. I'll remove it in a fixup PR.

Reply:
Oh, I didn't see this in detail.
It was wrapped twice.